docs/docs/migration.md

```diff
@@ -97,7 +114,20 @@ resources:
       path: ./xs-security.json
       config:
         xsappname: "${default-xsappname}"
+```
+
+ In the `xs-security.json` file, we  recommend to assign the property a temporary value that does not use the parameter. During deploymnet, the value specified directly in the MTA descriptor overrides the value specified in the `JSON` file.
```

Comment:
Why should we recommend this? I'd recommend to remove the property since it's specified in the mta.yaml...

Reply:
As a placeholder. But I will re-phrase it so it does not sound as a recommendation and also mention the option to remove.